### PR TITLE
Generic button label

### LINKED ISSUE #8: Create generic button and label

### PR DESCRIPTION
**Pull request by jenniferlim**
Issue fixed : https://github.com/jenniferlim/weTranslate/issues/8

### CODE REVIEW

**Comment by jenniferlim** [comment]
@lionel-alves -- Could you mark the issue as closed when the PR is merged.

File: weTranslate/Button.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//
+
+import UIKit
+
+final class Button: UIButton {
+    
+    // MARK: - Init
+    
+    init(title: String, color: UIColor) {
+
```
Comment (lionel-alves) [inline weTranslate/Button.swift]: :scissors:

File: weTranslate/Button.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import UIKit
+
+final class Button: UIButton {
+    
+    // MARK: - Init
+    
+    init(title: String, color: UIColor) {
+        
+        super.init(frame: CGRectZero)
+
```
Comment (lionel-alves) [inline weTranslate/Button.swift]: :scissors:

File: weTranslate/Label.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+//
+
+import UIKit
+
+final class Label: UILabel {
+
+    // MARK: - Init
+    
+    init(textColor: UIColor, font: UIFont?) {
+        super.init(frame: CGRectZero)
```
Comment (lionel-alves) [inline weTranslate/Label.swift]: CGRectZero  -> `.zero`

File: weTranslate/Label.swift [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        translatesAutoresizingMaskIntoConstraints = false
+        self.textColor = textColor
+        self.font = font
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
```
Comment (lionel-alves) [inline weTranslate/Label.swift]: :scissors:

File: weTranslate/TranslationView.swift [hunk 5]
```diff
@@ -21,22 +21,16 @@ final class TranslationView: UIView {
         }
     }
     
-    private let wordLabel: UILabel = {
-        let label = UILabel()
-        label.translatesAutoresizingMaskIntoConstraints = false
+    private let wordLabel: Label = {
+        let label = Label(textColor: UIColor(red: 247/255, green: 80/255, blue: 50/255, alpha: 1), font: UIFont(name: "HelveticaNeue", size: 36.0))
```
Comment (lionel-alves) [inline weTranslate/TranslationView.swift]: Can we use `Color` instead?

**Comment by lionel-alves** [comment]
A few comments but it looks :gucci:

**Comment by lionel-alves** [comment]
:bowtie:

**Pull request merged**